### DISCUSSION TITLE
portale solo ad uso interno

### DISCUSSION THREAD

**Discussion by dfilippi** [globaleaks/globaleaks-whistleblowing-software#3886] [General]
Ciao a tutti,
Nella mia azienda abbiamo appena iniziato ad usare questa piattaforma SOLO INTERNAMENTE, cioè gli utenti accederanno al wifi aziendale per poi accedere al portale.

C'è un modo per bypassare l'https in modo che non dia "connessione non sicura"?

**Comment by evilaliv3** [comment]
Premesso che a nosto avviso nel contesto normativo corrente una piattaforma per essere compliant deve essere pubblicata per essere accessibile dagli aventi diritto a segnalare (che potrebbero non avere accesso alla rete aziendale) ed essere fruita senza tracciamento (effettuato ad esempio dalla rete wifi aziendale), e la piattaforma non è connessa ad internet e non si possono acquisire i certificati letsencrypt potete comunque acquistare un certificato HTTPS e risolvere il problema di connessione non sicura.

**Comment by spomar** [comment]
ci sono tanti modi diversi, piu' o meno macchinosi ma funzionali.
Ad sempio puoi generare un certificato letsencrypt per un host locale (usando certbot via dns) che poi metterai nel tuo dns aziendale
Il tuo webserver aziendale fara' proxy verso il servizio globaleaks usando quel crt
Oppure metti globaleaks e webserver in vpn e sempre via proxy li fai parlare usando il crt ssl.
